docs/website/static/scripts/verify_signer_registration.sh

```diff
@@ -2,13 +2,16 @@
 
 set -e
 
-if [ -z "$SIGNER_LOGS_PATH" ]; then
-    echo ">> ERROR: Required environment variable SIGNER_LOGS_PATH is not set."
+if [ -z "$AGGREGATOR_ENDPOINT" ] || [ -z "$PARTY_ID" ]; then
+    echo ">> ERROR: Required environment variables AGGREGATOR_ENDPOINT and/or PARTY_ID are not set."
     exit 1
 fi
 
-if grep -q "STATE MACHINE: new cycle: Registered" "$SIGNER_LOGS_PATH"; then
+CURRENT_EPOCH=$(curl -s "$AGGREGATOR_ENDPOINT/epoch-settings" -H 'accept: application/json' | jq -r '.epoch')
+SIGNERS_REGISTERED_RESPONSE=$(curl -s "$AGGREGATOR_ENDPOINT/signers/registered/$CURRENT_EPOCH" -H 'accept: application/json')
+
+if echo "$SIGNERS_REGISTERED_RESPONSE" | grep -q "$PARTY_ID"; then
     echo ">> Congrats, your signer node is registered!"
 else
-    echo ">> Oops, your signer node is not registered. Check your configuration."
+    echo ">> Oops, your signer node is not registered. Party ID not found among the signers registered at epoch $((CURRENT_EPOCH - 1))."
```

Comment:
Why `- 1` ?
This route give the registrations for the given epoch (meaning that still under way if queried for the current cardano chain epoch), the one that will be signed into the certificates chain at `epoch + 1` to be able to issue single signatures at `epoch + 2`.

Reply:
My bad, it was a misunderstanding. 
I'll correct from `CURRENT_EPOCH - 1` to `CURRENT_EPOCH`.